wiz-broker/templates/secrets.yaml

```diff
@@ -7,12 +7,6 @@ metadata:
   namespace: {{ .Release.Namespace | quote }}
   labels:
     {{- include "wiz-broker.labels" . | nindent 4 }}
-  annotations:
-    "helm.sh/hook": pre-install,pre-upgrade
-    "helm.sh/hook-weight": "-1"
-    {{- with .Values.global.wizConnector.annotations }}
```

Comment:
You need the option to add annotations by global values, delete just "helm.sh/hook" and "helm.sh/hook-weight" keys